llm/server.go

```diff
@@ -244,6 +284,8 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 		params = append(params, "--flash-attn")
 	}
 
+	setCacheTypeParams(&params, flashAttnEnabled, opts)
```

Comment:
setCacheTypeParams could most likely be put inline here to avoid passing in `&params`.

Reply:
Done!